client/dashboard/sites/settings/index.tsx

```diff
@@ -24,11 +24,10 @@ export default function SiteSettings() {
 			<Heading>{ __( 'General' ) }</Heading>
 			<Card>
 				<VStack>
-					<RouterLinkSummaryButton
-						to={ `/sites/${ siteSlug }/settings/subscription-gifting` }
-						title={ __( 'Accept a gift subscription' ) }
-						density="medium"
-						badges={ getSubscriptionGiftingSettingBadges( settings ) }
+					<SubscriptionGiftingSettingsSummary
```

Comment:
I ended up bringing back this summary component. Now it contains logic, so I wanted to decouple it from here.

In the future, we will also have custom logic like upsells, which depend on the site, so it won't be just returning `<RouterLinkSummaryButton/>` anyway.